server/lib/nl/fulfillment/basic.py

```diff
@@ -133,9 +133,9 @@ def _populate_explore(state: PopulateState, chart_vars: ChartVars,
   return added
 
 
-def _populate_chat(state: PopulateState, chart_vars: ChartVars,
-                   places: List[Place], chart_origin: ChartOriginType,
-                   rank: int) -> bool:
+def _populate_chart(state: PopulateState, chart_vars: ChartVars,
```

Comment:
I think it was "chat" vs. "explore" :)

Reply:
ohh, got it, thought it was a typo, will revert

Reply:
done